frontend/forms/mediaUpload/MediaUpload.tsx

```diff
@@ -0,0 +1,142 @@
+import { ChangeEvent, FunctionComponent, useState } from "react";
+import { Stack } from "@chakra-ui/layout";
+import { FilePicker, FilePickerProps } from "../../inputs/filePicker";
+import { Accept } from "react-dropzone";
+import {
+  Flex,
+  FormControl,
+  FormErrorMessage,
+  FormHelperText,
+  FormLabel,
+  Heading,
+  HStack,
+  Switch,
+  VStack,
+} from "@chakra-ui/react";
+import { LanguageInput } from "../../inputs/languageInput";
+import { isArray } from "lodash";
+
+export interface TranscribeProps {
+  languages: string[];
+  enablePiiRedaction: boolean;
+}
+
+export interface MediaUploadProps {
+  onSubmit: (transcribeProps: TranscribeProps, files: File[]) => void;
+}
+
+const SUPPORTED_MIME_TYPES = [
+  "audio/flac",
+  "audio/mpeg",
+  "audio/mp4",
+  "video/mp4",
+  "audio/m4a",
+  "audio/x-m4a",
+  "application/ogg",
+  "audio/ogg",
+  "video/ogg",
+  "video/webm",
+  "audio/webm",
+  "audio/amr",
+  "audio/x-wav",
+  "audio/vnd.wave",
+  "audio/wav",
+  "audio/wave",
+  "audio/x-pn-wav",
+];
+
+const accept: Accept = {}; // specify the type of accept
+SUPPORTED_MIME_TYPES.forEach((mimeType) => {
+  accept[mimeType] = [];
+});
+
+export const MediaUpload: FunctionComponent<MediaUploadProps> = ({
+  onSubmit,
+}) => {
+  const [languages, setLanguages] = useState<string[]>(["en-AU"]);
+  const [enablePiiRedaction, setEnablePiiRedaction] = useState<boolean>(false);
+
+  const onLanguageChange = (selectedLanguages: string | string[]) => {
+    setLanguages(
+      isArray(selectedLanguages) ? selectedLanguages : [selectedLanguages],
+    );
+  };
+
+  const onEnablePiiRedactionChange = (event: ChangeEvent<HTMLInputElement>) => {
+    const isChecked = event.target.checked;
+    if (isChecked) {
+      setLanguages(["en-US"]);
+    }
+    setEnablePiiRedaction(isChecked);
+  };
+
+  const onFilesPicked = (files: File[]) => {
+    onSubmit({ languages, enablePiiRedaction }, files);
+  };
+
+  const filePickerProps: FilePickerProps = {
+    accept,
+    maxSize: 2 * 1024 * 1024 * 1024, // 2GB
+    maxDuration: { hours: 4 },
+    storageDuration: { days: 12 },
```

Comment:
14, according to https://github.com/eresearchqut/transcription/blob/32848b1c5c63d915f15e3607c3823e140a22f92d/api/src/repository/repository.ts#L14